src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -124,19 +124,20 @@ private async Task createReportConfigurationPlan(QualificationRunResult[] runRes
          reportConfigurationPlan.SimulationMappings = toJArray(mappings.SelectMany(x => x.SimulationMappings));
 
          reportConfigurationPlan.ObservededDataSets = toJArray(mappings.SelectMany(x => x.ObservedDataMappings).Union(staticFiles.ObservedDatSets));
-
+         
          var plots = qualificationPlan.Plots;
          RemoveByName(plots, Configuration.ALL_PLOTS);
 
          plots.TimeProfile = toJArray(mappings.SelectMany(x => x.Plots));
          reportConfigurationPlan.Plots = plots;
 
+         reportConfigurationPlan.Inputs = toJArray(mappings.SelectMany(x => x.Inputs));

```

Comment:
This is how you can add a dynamic content to a dynamic array. I am using a JObject directly (kind of break the encapsulation) but this is so much easier in that case because we are exporting Json. All the dynamic stuff is working out of the box